File: 1431-kids-with-the-greatest-number-of-candies/1431-kids-with-the-greatest-number-of-candies.ts
function kidsWithCandies(candies: number[], extraCandies: number): boolean[] {
    let result =[];
    let max=Math.max(...candies)
    for(let i=0;i<candies.length;i++){
        if(candies[i]+extraCandies>=max){
            result.push(true)
        }else{
            result.push(false)
        }
    }
    return result;
};

File: 1431-kids-with-the-greatest-number-of-candies/1431-kids-with-the-greatest-number-of-candies.cs
public class Solution {
    public IList<bool> KidsWithCandies(int[] candies, int extraCandies) {
       int max=candies.Max();
       List<bool> result=new List<bool>();
       for(int i=0;i<candies.Length;i++){
        if(candies[i]+extraCandies>=max){
            result.Add(true);
        }else{
            result.Add(false);
        }
       } 
       return result;
    }
}